docs/pages/datapoint.js

```diff
@@ -57,11 +66,9 @@ export default function DatapointPage({ generatedDocGen }: {| generatedDocGen: D
 <Datapoint size="lg" title="Saves" value="10,392" trend={{value: -12.193, accessibilityLabel: "Trending down"}} />
 `}
           />
-          {/*
-            The example below should also display a localized trend value, but given trend accepts only a number,
-            the value cannot be localized. Once the API is changed, we should add a localized trend to this example.
-            */}
           <MainSection.Card
+            // This example should also display a localized trend value, but given trend accepts only a number,
```

Comment:
moving this comment here prevents prettier from collapsing whitespace used to break up the markup